app/src/main/java/com/humberto/tasky/agenda/presentation/edit_text/EditTextScreen.kt

```diff
@@ -0,0 +1,168 @@
+package com.humberto.tasky.agenda.presentation.edit_text
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.text.BasicTextField
+import androidx.compose.foundation.text.KeyboardOptions
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.ChevronLeft
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.Icon
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.focus.FocusRequester
+import androidx.compose.ui.focus.focusRequester
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.input.ImeAction
+import androidx.compose.ui.text.input.KeyboardType
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.hilt.navigation.compose.hiltViewModel
+import androidx.lifecycle.compose.collectAsStateWithLifecycle
+import com.humberto.tasky.R
+import com.humberto.tasky.core.presentation.designsystem.TaskyGreen
+import com.humberto.tasky.core.presentation.designsystem.TaskyTheme
+import com.humberto.tasky.core.presentation.designsystem.components.TaskyScaffold
+import com.humberto.tasky.core.presentation.designsystem.components.TaskyToolbar
+import com.humberto.tasky.main.navigation.EditTextScreen
+import timber.log.Timber
+
+@Composable
+fun EditTextScreenRoot(
+    onGoBack: () -> Unit,
+    onSaveClick: (EditTextScreen) -> Unit,
```

Comment:
This is a confusing name, considering that you also use `EditTextScreen` for a composable. You also use the same naming convention throughout your project (`Screen` suffix) for your composable screens. Do you think a different name would be better here?